tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js

```diff
@@ -52,12 +52,14 @@ module.exports = {
      * inside the collaborator element, defaults to all when null
      * @param filterDisplayName
      * @param timeout
+     * @param expectDisplaynameBePresent
      * @returns {Promise.<string[]>} Array of users/groups in share list
      */
     getCollaboratorsList: async function(
       subSelectors = null,
       filterDisplayName = null,
-      timeout = null
+      timeout = null,
+      expectDisplaynameBePresent = true
```

Comment:
IMO, this is not needed. simply call this function like
```js
getCollaboratorsList(
    { displayName: collaboratorsDialog.elements.collaboratorInformationSubName},
    null,
    client.globals.waitForNegativeConditionTimeout
)
```

Reply:
get all the collaborators and just assert that given `<name>` is not present in the list.

Reply:
Also refactor the following block
https://github.com/owncloud/web/blob/254c568df6adfeb6ccba6022d051956b7eef4e7d/tests/acceptance/stepDefinitions/sharingContext.js#L337
to 
```js
return (name === share.displayName && (!share.shareType || type === share.shareType.toLowerCase()))
```
because `share` doesn't have property `shareType` in some cases (at least when I tried)
I got following:
```js
[
  { displayName: 'Alice Hansen' },
  { displayName: 'Carol King' },
  { displayName: 'grp2' }
]
```